packages/pasteup/package.json

```diff
@@ -1,16 +1,23 @@
 {
   "name": "@guardian/pasteup",
-  "version": "0.1.0-alpha.1",
+  "version": "1.0.0-alpha.1",
   "description": "Guardian design tokens",
-  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/guardian/dotcom-rendering"
+  },
+  "files": [
+    "breakpoints.js",
+    "fonts.js",
+    "mixins.js",
+    "palette.js",
+    "icons",
+    "logos"
+  ],
   "scripts": {
-    "test": "jest"
+    "test": "jest",
```

Comment:
What do you think about adding a yarn script to call `make clean-pasteup`? 

Currently if a developer's making changes to `pasteup` and they run `yarn postpublish` but then make a change before publishing they'd have to step back to the `dotcom-rendering` directory and run `make clean-pasteup` before they can run `yarn postpublish` again. (it'd error if they tried without cleaning up right now if `src` already exists).

Reply:
@ghaberis I dunno, the prepublish and postpublish hooks are not intended to be used by devs, they are automatically executed before and after yarn publish, respectively